src/data/sample-activity-1.json

```diff
@@ -4,198 +4,198 @@
   "layout": 0,
   "name": "Single Page Test Activity",
   "notes": "",
-  "project": null,
+  "project": {
```

Comment:
I am wondering if this `sample-activity-1.json` file is in the new format, or the old format, and whether or not that matters.

I noticed that a number of the tests were changed to load data from the `../data/version-2/` path, which made sense. But now I am wondering, was this activity changed/updated in place by accident?

Reply:
Since we can now convert documents on the fly, I reverted sample-activity-1 to its original form (legacy LARA).
Jest tests point to the version-2 activities because the conversion doesn't happen on the fly.